akka-stream-tests/src/test/scala/akka/stream/scaladsl/LazilyAsyncSpec.scala

```diff
@@ -0,0 +1,77 @@
+package akka.stream.scaladsl
```

Comment:
All files must have the header:

```
/**
 * Copyright (C) 2014-2018 Lightbend Inc. <https://www.lightbend.com>
 */
```

We'll soon automate that I hope

Reply:
Added but also see that you did that already, sorry for kind of a mess :(

Reply:
Oh, sorry -- I somehow expected you be in EU timezone so was taking my timeprepping and going to merge this while EU is asleep :)

Reply:
CST actually :) but thanks for taking care of it, much appreciated

Reply:
Also, I could work on automating copyright check if you can supply issue for it

Reply:
Cheers to CST from JST then :)

For the copyright we want to replicate what we have here: https://github.com/akka/akka-http/pull/1738